samples/bluetooth/iso_combined_bis_and_cis/README.rst

```diff
@@ -0,0 +1,120 @@
+.. _bluetooth_isochronous_combined_bis_and_cis:
+
+Bluetooth: ISO Combined BIS and CIS
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® Combined BIS and CIS sample demonstrates capability of
+a Controller to support both Broadcast Isochronous Stream (BIS) and Connected Isochronous Stream (CIS) and
+can be used to learn ISO-specific APIs to create stream, receive and send data over either CIS or BIS.
+Note: the sample uses the simplest way to send data over ISO without using timestamps through a single stream,
+therefore it doesn't demonstrate any capabilities of synchronization multiple streams.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+You can use any of the development kits listed above and mix different development kits.
+
+.. include:: /includes/hci_ipc_overlay.txt
+
+The sample also requires a connection to a computer with a serial terminal |ANSI| for each of the development kits.
+
+Overview
+********
+
+The sample demonstrates a device combining CIS Central and BIS Source.
```

Comment:
I'm missing a description of what the sample actually does. I propose to move the description from line 84ff up here.

Reply:
Thank you, I added a very short description here.
Do you think it is ok now, or it makes sense to provide more detailed explanation?